core/statistics.c

```diff
@@ -104,7 +108,7 @@ void calculate_stats_summary(struct stats_summary *out, bool selected_only)
 	int prev_month = 0, prev_year = 0;
 	int trip_iter = 0;
 	dive_trip_t *trip_ptr = 0;
-	unsigned int size, tsize;
+	unsigned int size, tsize, dsize, tmsize;
```

Comment:
Idle comment: It's in the original code, but "unsigned int" seems like a strange choice to me. `size_t` [type of `sizeof(x)`] or `int` would be more logical to me.